bosshud/ze_castlevania_64_v1_3_b9_go.txt

```diff
@@ -0,0 +1,13 @@
+"math_counter"
+{
+	"0"
+	{
+		"HP_counter"		"BossCounter1"
+		"HPbar_counter"		"BossCounter3"
+		"HPinit_counter"	"BossCounter2"
+		"HPbar_min"		"0"
+		"HPbar_max"		"20"
+		"HPbar_default"		"20"
+		"HPbar_mode"		"1"
+	}
+}
```

Comment:
Castlevania_64 can't have a bhud config due to the maps usage of these HP entities for many other non-boss related things. It was previously removed for this reason already.